shell/styles/index.css

```diff
@@ -80,3 +80,25 @@ footer .flex-row > * {
 span.hljs-comment {
   color: #a69685 !important;
 }
+
+blockquote {
+  color: #7a7a7a;
+  font-style: normal;
+  background-color: #1f1f1f;
+  border-radius: 4px;
+  border-left: 9px solid #654ff0;
+  padding: 20px 45px 20px 26px;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.15);
+}
+
+blockquote > .label {
+  font-weight: 700;
+  font-size: 0.8em;
+  color: #654ff0;
```

Comment:
Can we make this color lighter? I think it will be hard to see for accessibility needs users. You can see the contrast ratio by using the color picker in dev tools:

![Screenshot from 2019-08-13 01-14-27](https://user-images.githubusercontent.com/1448289/62925690-beb00a80-bd67-11e9-9bd8-95b17e07a6e6.png)

Reply:
Can't find how to enable contrast ratio in dev tools

but is it ok now?
![image](https://user-images.githubusercontent.com/4308648/62962963-023d5f80-bdce-11e9-844b-aa160cd1554c.png)

Reply:
Sorry for getting back to this so late! Would you mind making it a little lighter?

Something like: #c2b3ea or #747b8c would be good for the color  :smile:

Reply:
I liked `#747b8c`, but it has less contrast:

![image](https://user-images.githubusercontent.com/4308648/63214474-a54feb00-c0e6-11e9-9231-12269a7e08a9.png)

`#c2b3ea` in the other hand has better contrast:
![image](https://user-images.githubusercontent.com/4308648/63214518-07a8eb80-c0e7-11e9-9ab1-4f470815ab7e.png)


So let's stay with `#c2b3ea`.